cluster-provision/gocli/cmd/run.go

```diff
@@ -475,6 +477,23 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			}
 		}
 
+		var vmArgsUSBDisks []string
+		const bus = " -device qemu-xhci,id=bus%d"
+		const drive = " -drive if=none,id=stick%d,format=raw,file=/usb-%d.img"
+		const dev = " -device usb-storage,bus=bus%d.0,drive=stick%d"
```

Comment:
Didn't test but [documentation](https://www.qemu.org/docs/master/system/devices/usb.html) says we don't need to set `bus=` when using qemu-xhci

Reply:
Depends on # of controllers afaik.